packages/builder/src/internal/cli/cli.ts

```diff
@@ -132,10 +127,9 @@ async function main() {
       );
     }
 
-    // TODO: This is here for backwards compatibility
-    // There are very few projects using this.
-    if (builderArguments.network === undefined) {
-      builderArguments.network = config.defaultNetwork;
+    if (!builderArguments.network) {
+      // TODO:RZ
+      throw new Error("INTERNAL ERROR. Default network should be registered in `register.ts` module")
```

Comment:
You should reuse the pattern of `BuilderError`. It is printed out in console. You can add a message and provide the parameters for formatting.

Reply:
I intentionally didn't want it - I want to remove this Errors later: https://www.pivotaltracker.com/story/show/173584552

Reply:
Ah, it's the `TODO:RZ` comment. We also have the default config somewhere in the project. This config already has some details and you should simply replace the arguments and crash if no network is found.

The user will not know how to recover from deleted config if he accidentally deletes the `default: {}` section.
Do we want the user to recover from this?